### DISCUSSION TITLE
Subscriptions support

### DISCUSSION THREAD

**Discussion by Andrey-Bazhanov** [keystonejs/keystone#6567] [Event Q&A Q3 2021]
Hi, thanks for awesome project!

Do you have any plans for support GraphQL subscriptions in future?

**Comment by raveling** [comment]
Thanks for your question @Andrey-Bazhanov 
We answered this in the recent community Q&A event: 
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=1672s
